File: my-code/344/main.py
# -*- coding:utf-8 -*-
# @Time     : 2021/07/29 00:27
# @Author   : Ranshi
# @File     : main.py
from typing import List


class Solution:
    def reverseString(self, s: List[str]) -> None:
        s.reverse()


if __name__ == '__main__':
    s = ["h", "e", "l", "l", "o"]
    print(Solution().reverseString(s))
    print(s)

File: my-code/344/main.go
package main

// @Time     : 2021/5/31 10:14
// @Author   : Ranshi
// @File     : 344. 反转字符串.go

func reverseString(s []byte) {
	for i, j := 0, len(s)-1; i < j; i, j = i+1, j-1 {
		s[i], s[j] = s[j], s[i]
	}
}

func main() {
}
